get_number.rb

```diff
@@ -0,0 +1,48 @@
+# Get My Number Game
```

Comment:
There could be a shebang line
https://www.google.com/search?q=shebang+line+ruby&oq=shebang+line&aqs=chrome.1.69i57j0l7.5222j0j7&sourceid=chrome&ie=UTF-8